pages/wiki/ip-connection.md

```diff
@@ -0,0 +1,49 @@
+---
+title: IP Connection
+layout: documentation
+---
+
+Configuring an IP connection on your watch has to be done manually until a GUI settings option is available.
+On watches with supported WLan, you can enable wifi and configure the connection using <code>connmanctl</code> like described below.
+Forwarding IP requests to a connected PC via USB is another option, explained in the second paragraph.
+
+By default, there is no <code>root</code> or <code>ceres</code> password, and no firewall rules. A password can be set using the <code>passwd</code> command.
+
+
+
+# IP over WLan
```

Comment:
This seems to always be written in all caps: WLAN https://en.m.wikipedia.org/wiki/Wireless_LAN (same comment for the other pages)
But I also alway had the feeling that this is a German word. In France for example no one understands what WLAN means, everybody talks about WiFi instead.

Reply:
WLAN is simply a LAN that uses radio technology instead of wiring to interconnect networked nodes. Which also includes Ad-hoc or completely proprietary implementations.
Wi-Fi is the wireless standard 802.11 and nothing else. 

WLAN not being generally understood is a shame imo and introduced the confusion we now have with 5G. Which is also just a WLAN by definition.
WLan is wrong and rooted in gamer community wanting to let the word look fancy like WiFi :D

I agree, we solely have Wi-Fi on our watches. I was coming from SEO perspective and tried to drop the WLAN term just for the search engines in case users search for "WLAN on Asteroid, etc" from outside.

Reply:
Also going for Wi-Fi instead if WiFi. 
WiFi makes sense from SEO perspective since it is top google choice. But all publications i trust use Wi-Fi officially.
Plus Wifi is already SEO covered by the `wifi` mentions we have.